tests/snuba/api/endpoints/test_organization_events.py

```diff
@@ -2000,14 +2001,16 @@ def test_user_misery_alias_field_with_project_threshold(self):
         assert data[0]["user_misery()"] == pytest.approx(0.05751, rel=1e-3)
         assert data[1]["user_misery()"] == pytest.approx(0.06586, rel=1e-3)
 
-    def test_user_misery_alias_field_with_transaction_threshold(self):
+    def _setup_user_misery(self, per_transaction_threshold: bool = False) -> None:
+        # If duration is > 300 * 4 then the user is fruistrated
+        # There's a total of 4 users and three of them reach the frustration threshold
         events = [
```

Comment:
In the `mep` test module you will see that we try to iterate in a similar manner to a data structure like this one.
We may need to move it somewhere to be shared and avoid duplication.